vnext/Shared/Modules/WebSocketModule.cpp

```diff
@@ -487,6 +493,29 @@ void WebSocketTurboModule::RemoveListeners(double /*count*/) noexcept {}
 
 #pragma endregion WebSocketTurboModule
 
+#pragma region WebSocketTurboModuleProxy
+
+WebSocketTurboModuleProxy::WebSocketTurboModuleProxy(
+    std::unordered_map<double, shared_ptr<IWebSocketResource>> &resourceMap) noexcept
+    : m_resourceMap{resourceMap} {}
+
+#pragma endregion WebSocketTurboModuleProxy
+
+void WebSocketTurboModuleProxy::SendBinary(string &&base64String, int64_t id) noexcept /*override*/
+{
+  auto rcItr = m_resourceMap.find(static_cast<double>(id));
+  if (rcItr == m_resourceMap.cend()) {
+    return;
+  }
+
+  weak_ptr<IWebSocketResource> weakRc = (*rcItr).second;
+  if (auto rc = weakRc.lock()) {
+    rc->SendBinary(std::move(base64String));
+  }
```

Comment:
Should there be any indication of the method not doing what it's supposed to?

Reply:
Not following.
Do you mean an error condition?
I believe the `SendBinary` implementation handles those.

Reply:
Other platforms have a circular dependency between Blob,WS and Blob,HTTP modules.

They do not account for the case where they wouldn't be available, likely because this would only happen on instance tear-down.

This design is less coupled, but ultimately the proxy object is owned by the WebSocket module so the same semantics should apply.